source/guides/puppetlabs_package_verification.markdown

```diff
@@ -35,31 +35,32 @@ Before you can verify signatures, you must import the Puppet Labs public key and
 
 To import the release signing key, run:
 
-    $ gpg --recv-key 1054b7a24bd6ec30
-    gpg: requesting key 4BD6EC30 from hkp server pool.sks-keyservers.net
-    gpg: key 4BD6EC30: public key "Puppet Labs Release Key " imported
+    $ gpg --recv-key 7F438280EF8D349F
```

Comment:
Do we want to specify the `pgp.mit.edu` keyserver here, with `--keyserver pgp.mit.edu`?